src/Tribe/Linked_Posts.php

```diff
@@ -793,93 +851,54 @@ public function saved_linked_post_dropdown( $post_type, $current = null ) {
-				}
+		// If we have both we make it an array
+		} else {
+			$data = array_values( (array) $options );
+		}
 
-				/**
-				 * Filters the linked post dropdown optgroup label that holds organizers that have
-				 * been created by that user
-				 *
-				 * @param string $my_optgroup_name Label of the optgroup for the "My X" section
-				 * @param string $post_type Post type of the linked post
-				 */
-				$my_optgroup_name = apply_filters( 'tribe_events_saved_linked_post_dropdown_my_optgroup', $my_optgroup_name, $post_type );
-
-				echo $linked_posts ? '<optgroup label="' . esc_attr( $my_optgroup_name ) . '">' : '';
-				echo $my_linked_post_options;
-				echo $linked_posts ? '</optgroup>' : '';
-			}
+		/*
 
-			if ( $linked_posts ) {
-				$optgroup_name = sprintf( esc_html__( 'Available %s', 'the-events-calendar' ), $plural_name );
-
-				// backwards compatibility with old organizer filter
-				if ( Tribe__Events__Organizer::POSTTYPE === $post_type ) {
-					/**
-					 * Filters the linked organizer dropdown optgroup label for saved organizers
-					 *
-					 * @deprecated 4.2
-					 *
-					 * @param string $my_optgroup_name Label of the optgroup for the "Available Organizers" section
-					 */
-					$optgroup_name = apply_filters( 'tribe_events_saved_organizers_dropdown_optgroup', $optgroup_name );
-				}
+		! empty( $post_type_object->cap->create_posts )
+		&& current_user_can( $post_type_object->cap->create_posts )
+		&& ! empty( $this->linked_post_types[ $post_type ]['allow_creation'] )
 
-				/**
-				 * Filters the linked post dropdown optgroup label that holds all published posts of the given type
-				 *
-				 * @param string $my_optgroup_name Label of the optgroup for the "Available X" section
-				 * @param string $post_type Post type of the linked post
-				 */
-				$optgroup_name = apply_filters( 'tribe_events_saved_linked_post_dropdown_optgroup', $optgroup_name, $post_type );
-
-				echo $my_linked_posts ? '<optgroup label="' . esc_attr( $optgroup_name ) . '">' : '';
-				foreach ( $linked_posts as $linked_post ) {
-					$linked_post_title = wp_kses( get_the_title( $linked_post->ID ), array() );
-					echo '<option value="' . esc_attr( $linked_post->ID ) . '"';
-					selected( $current == $linked_post->ID );
-					echo '>' . $linked_post_title . '</option>';
-				}
-				echo $my_linked_posts ? '</optgroup>' : '';
-			}
-			echo '</select>';
+		 */
+
+		if ( $linked_posts || $my_linked_posts ) {
+			echo '<input
+				type="hidden"
+				class="tribe-dropdown linked-post-dropdown"
+				name="' . esc_attr( $name ) . '"
+				id="saved_' . esc_attr( $post_type ) . '"
+				data-freeform
+				data-placeholder="' . sprintf( esc_html__( 'Create or Select a %s', 'the-events-calendar' ), $singular_name ) . '"
+				data-search-placeholder="' . sprintf( esc_html__( 'Search or Create a new %s', 'the-events-calendar' ), $singular_name ) . '"
```

Comment:
This (and the line above it) should probably have `esc_attr__()` rather than `esc_html__()`